src/components/Photos/Photos.js

```diff
@@ -29,19 +36,21 @@ class Photos extends React.Component {
 
     return (
       <>
+        <LoadingBar color="#6958f2" ref={this.loadingBar} />
         {photos && photos.length === 0 && <div>This user has no photos</div>}
```

Comment:
I like saving this checks in variables and give them a readable name

Reply:
Good point. I will refactor.